pkg/registry/core/node/rest/proxy.go

```diff
@@ -45,9 +45,9 @@ var _ = rest.Connecter(&ProxyREST{})
 
 var proxyMethods = []string{"GET", "POST", "PUT", "PATCH", "DELETE", "HEAD", "OPTIONS"}
 
-// New returns an empty nodeProxyOptions object.
+// New returns an empty node object.
 func (r *ProxyREST) New() runtime.Object {
-	return &api.NodeProxyOptions{}
+	return &api.Node{}
```

Comment:
According to https://github.com/kubernetes/kubernetes/pull/66807/commits/64bd0e45894bbd97fe0c455a7d3350f5be5d15da: 
> The kind is used by the admission webhook controller to
determine how to convert the object

changing these kinds will break the admission webhook controller.